api/tests/__init__.py

```diff
@@ -17,3 +18,5 @@
 
 settings.CELERY_ALWAYS_EAGER = True
 settings.WC_WEBHOOK_USER = 'test_user'
+
+os.environ['REDIS_URL'] = 'rediss://test'
```

Comment:
We'll have to review the way we handle settings for testing again. This is going astray :see_no_evil: